lms/views/admin/lti_registration.py

```diff
@@ -240,3 +235,9 @@ def _get_url_suggestions(issuer, client_id):
                 }
 
         return {"auth_login_url": None, "key_set_url": None, "token_url": None}
+
+    def _get_registration_or_404(self, id_) -> LTIRegistration:
+        if org := self.lti_registration_service.get_by_id(id_):
+            return org
+
+        raise HTTPNotFound()
```

Comment:
This isn't very fancy, but users shouldn't be typing in URLs by hand, so this seems fine.